### ISSUE TITLE
Unable to flash RM0329-0318-0.100.2-f601860 through the web installer

### ISSUE THREAD

**Issue by strasharo** [RogueMaster/flipperzero-firmware-wPlugins#1051]
### Describe the bug.

Flashing using the web installer from the link https://lab.flipper.net/?url=https%3A%2F%2Frogue-master.net%2F%3Ffile%3DRM0329-0318-0.100.2-f601860.tgz&channel=RM0329-0318-0.100.2-f601860&version=0.100.2 fails with
![image](https://github.com/RogueMaster/flipperzero-firmware-wPlugins/assets/3337997/63015568-d9ab-4212-8b65-75fedecadf37)

### Reproduction

1. Connect the Fliper to the computer
2. Open in Chromium https://lab.flipper.net/?url=https%3A%2F%2Frogue-master.net%2F%3Ffile%3DRM0329-0318-0.100.2-f601860.tgz&channel=RM0329-0318-0.100.2-f601860&version=0.100.2
3. Press Install

### Target

RM0329-0318-0.100.2-f601860

### Logs

```Text
Failed to fetch firmware: TypeError: Cannot read properties of undefined (reading 'data')
TypeError: Cannot read properties of undefined (reading 'data')
```

### Anything else?

Was upgrading fine until now on all previous releases.
Worked fine through qFlipper.

**Comment by RogueMaster** [comment]
Will check into this issue

**Comment by RogueMaster** [comment]
Please use file from GitHub in the meantime

**Comment by gades** [comment]
> Please use file from GitHub in the meantime

it was not fixed in RM0329-0318-0.100.2-f601860
